### PR TITLE
additional clarity for getting started + contributing

### PR DESCRIPTION
**Pull request by ProsperousHeart**
For those who may be new, providing additional, explicit clarity on the different sections for getting started vs contributing

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,9 +70,10 @@ For Enterprise
 Getting Started
 ---------------
 
-See the `Quick Start section`_ of the docs.
+Looking to run ``coverage`` on your current `pytest`_ suite(s)? See the `Quick Start section`_ of the docs.
```
Comment (nedbat) [inline README.rst]:
```suggestion
Looking to run ``coverage`` on your test suite? See the `Quick Start section`_
of the docs.
```
(Coverage isn't particular to pytest)

File: README.rst [hunk 2]
```diff
@@ -70,9 +70,10 @@ For Enterprise
...
 
 .. _Quick Start section: https://coverage.readthedocs.io/#quick-start
+.. _pytest: https://docs.pytest.org
```
Comment (nedbat) [inline README.rst]: With my change above, we don't need this.

File: README.rst [hunk 3]
```diff
@@ -96,6 +97,8 @@ Community Code of Conduct`_.
 Contributing
 ------------
 
+Found a bug? Want to help improve the code or documentation?
+
 See the `Contributing section`_ of the docs.
```
Comment (nedbat) [inline README.rst]: Combine the two sentences in one paragraph?
```suggestion
Found a bug? Want to help improve the code or documentation? See the
`Contributing section`_ of the docs.
```

File: README.rst (hunk 1 above)
Comment (ProsperousHeart) [reply README.rst]: Ah! Makes sense.

File: README.rst (hunk 2 above)
Comment (nedbat) [inline README.rst]:
```suggestion
```

**Review by nedbat — APPROVED** [review]

**Pull request merged**